docs/meeting-notes.rst

```diff
@@ -0,0 +1,8 @@
+.. _meeting-notes:
+
+Pangeo Weekly Meeting Notes
+===========================
```

Comment:
Since this meeting is open to anyone, it may be worth putting the schedule and call in details here. Then we can close the GH issue (#391).